torch_xla/csrc/convolution.cpp

```diff
@@ -1,16 +1,34 @@
 #include "torch_xla/csrc/convolution.h"
 
-#include "tensorflow/compiler/tf2xla/kernels/conv_op_helpers.h"
-#include "tensorflow/compiler/xla/client/lib/constants.h"
-#include "tensorflow/core/framework/tensor_shape.h"
-#include "tensorflow/core/kernels/conv_grad_ops.h"
+#include "third_party/xla_client/conv_op_helpers.h"
+#include "xla/client/lib/constants.h"
+// #include "tensorflow/core/framework/tensor_shape.h"
+// #include "tensorflow/core/kernels/conv_grad_ops.h"
```

Comment:
What's the replacement for this line in openxla @ManfeiBai?

Reply:
Thanks, @miladm ,

not found replacement from OpenXLA